@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -85,9 +85,9 @@ const USER_INPUTS_TO_CONSIDER = 5;
 const DEFAULT_MAX_PITCH_ANGLE = Math.PI / 4;
 
 /**
- * Epsilon value to rule out when a number can be considered 0 (used for pitch angle in radian).
+ * Epsilon value to rule out when a number can be considered 0.
  */
-const EPSILON_PITCH = 0.01;
+const EPSILON = 0.01;
```

Comment:
This is an epsilon in radian used to define when toggling the 2D/3D view should be tilted (if the camera is already almost vertical, the user will not expect the animation to go for the last missing epsilon to vertical, but rather to the fully tilted view, it is used for this specific problem). The ease method below interpolates unitless values, all between 0 and 1. Likely the typedoc is missing these information in the first place. This epsilon should be kept for the pitch, and another one created for the other purpose.

Reply:
I get your reasoning, but its a private constant and in both cases we can use the same value. If we want to define different values in the future, I'd split, but I don't see why it's better to use 2 specific epsilons with the same value rather than one generic one.

Reply:
I agree with Andres, if we need to different values we can split later.